docs/api/api_blueprint.apib

```diff
@@ -13682,6 +13682,8 @@ Create a new ticket using an event_id.
                         "is-hidden": "false",
                         "min-order": "1",
                         "max-order": "100"
+                        "is-check-in-restricted": "true",
+                        "auto-check-in-enabled": "false"
```

Comment:
You have to add it in all the instances of tickets. For eg. in the GET ticket, GET list of tickets, PATCH tickets, GET tickets under an event etc. etc.